sql/schema.install.sql

```diff
@@ -50,12 +50,20 @@ CREATE TABLE __beachfront__detection (
     FOREIGN KEY (job_id) REFERENCES __beachfront__job(job_id) ON DELETE CASCADE
 );
 
+CREATE TABLE __beachfront__user (
+    geoaxis_uid       VARCHAR(64)    PRIMARY KEY,
+    user_name         VARCHAR(64)    NOT NULL,
+    api_key           VARCHAR(255)   NOT NULL    UNIQUE,
+    created_on        TIMESTAMP      NOT NULL    DEFAULT CURRENT_TIMESTAMP
+);
+
 CREATE TABLE __beachfront__job_user (
     job_id            VARCHAR(64),
     user_id           VARCHAR(64),
 
     PRIMARY KEY (job_id, user_id),
     FOREIGN KEY (job_id) REFERENCES __beachfront__job(job_id) ON DELETE CASCADE
+--    FOREIGN KEY (user_id) REFERENCES __beachfront__user(geoaxis_uid) ON DELETE CASCADE
```

Comment:
Can you uncomment this code and rename `geoaxis_uid` to `user_id` for foreign key consistency?